### PR TITLE
Improve prefetch count for exculsive queues, imporving performance of…

### PR DESCRIPTION
**Pull request by HeinKalido**
… monitor endpoints

Currently the monitor endpoints cannot process the messages fast enough (even though they are idling, and simply discarding most of these messages, because there is not listening monitors). 
This is most likely due to prefetch count being too small (at 4). 
Rabbit's estimate is 10% utilisation, so making the prefetch a 100 should  allow them to move fast enough, without prefetching everything (which should not matter because they are the exclusive readers on these queues).

### CODE REVIEW

**Review by KeoabetsweMorake — APPROVED** [review]

File: exchange.go [hunk 1]
```diff
@@ -147,6 +149,7 @@ func (re *RabbitExchangeImpl) ReceiveFrom(name, exchangeType string, durable, au
 		RoutingKey: key,
 		AutoDelete: true,
 		Exclusive:  true,
+		Prefetch:   rabbitPrefetchForSingleQueue + runtime.NumCPU()*2,
```
Comment (raulincze) [inline exchange.go:152]: Shouldn't we parametrise this and send it from outside when calling ReceiveFrom? I really wish golang had default parameters...

Comment (raulincze) [reply exchange.go:152]: I know it would be a pain to add the extra parameter everywhere :(

Comment (HeinKalido) [reply exchange.go:152]: Default parameters is exactly the problem here, adding them to the language does not solve the problem :) 


So the issue is we call Consume which calls ReceiveFrom which calls Receive that does take is as a parameter (with a default of 2x NumCPU. 
I won't mind adding the parameter to ImplRabbitEventHandler, to pass it to Receive.

Comment (raulincze) [reply exchange.go]: I see!

**Review by raulincze — APPROVED** [review]

**Pull request merged**